packages/angular/common/src/directives/control-value-accessors/radio-value-accessor.ts

```diff
@@ -12,10 +12,50 @@ import { ValueAccessor } from './value-accessor';
 
 @Directive()
 export class RadioValueAccessorBaseDirective extends ValueAccessor {
+
+  private readonly MAX_PARENT_LEVELS = 10;
+  private readonly PARENT_TAG = 'ix-radio-group';
+
   constructor(injector: Injector, el: ElementRef) {
     super(injector, el);
   }
 
+  override ngAfterViewInit() {
+    super.ngAfterViewInit();
+    this.handleRadioGroupValueChange()
+  }
+
+  private handleRadioGroupValueChange(): void {
+    const currentElement = this.elementRef.nativeElement;
+    const parentElement = this.findRadioGroup(currentElement);
```

Comment:
Instead of crawling the DOM I think we can simply access the asscocitated form control.
Example code (not necessarily the final solution):
```
  this.getAssignedNgControl()?.valueChanges?.subscribe((value: string) => {
      this.writeValue(value);
  });
```